src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public class SetEnvironmentVariable
```

Comment:
I may have missed it, but should there be a test that launches a process with specific environment variables and then verifies that all such variables (and only those) were the ones that showed up in GetEnvironmentVariables?  This would be in addition to these tests that verify roundtripping with Set/Get.